docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -31,13 +27,39 @@ Run the following command:
   ```
 
 #### **Kyma Dashboard**
-1. Navigate to `Cluster Details`.
-2. Open the Istio module's configuration.
-3. Click **Edit**.
-4. Add the number of trusted proxies and click **Update**.
-   ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+1. Navigate to **Cluster Details** and select **Modify Modules**.
+2. Choose the Istio module and select **Edit**.
+3. In the `General` section, add the number of trusted proxies.
+4. Select **Save**.
 <!-- tabs:end -->
 
+### Configure Gateway External Traffic Policy in the Istio Custom Resource (GCP and Azure only)
+
+If you are using a GCP or Azure cluster, you must also set the **gatewayExternalTrafficPolicy** to `Local` in order to include the client's IP address in the XFF header. Skip this step if you're using a different cloud service provider.
+
```

Comment:
```suggestion

For production Deployments, it is strongly recommended to deploy an Ingress Gateway Pod to multiple nodes if you enable `externalTrafficPolicy: Local`. Otherwise, this creates a situation where only nodes with an active Ingress Gateway Pod are able to accept and distribute incoming NLB traffic to the rest of the cluster, creating potential ingress traffic bottlenecks and reduced internal load balancing capability, or even complete loss of ingress traffic incoming to the cluster if the subset of nodes with Ingress Gateway Pods goes down. See the source IP for Services with `Type=NodePort` for more information. For reference, see Istio [Network Load Balancer](https://istio.io/latest/docs/tasks/security/authorization/authz-ingress/#network) documentation.
Default Kyma Istio installation profile configures **PodAntiAffinity** to ensure that Ingress Gateway Pods are evenly spread across all Nodes. This guarantees that the above requirement is satisfied if your IngressGateway autoscaling configuration **minReplicas** is at least equal to the number of Nodes. You can configure autoscaling options in the Istio custom resource using **spec.config.components.ingressGateway.k8s.hpaSpec.minReplicas**.

> [!WARNING]
> Deploy an Ingress Gateway Pod to multiple nodes if you enable `externalTrafficPolicy: Local` in production Deployments.
```